index.js

```diff
@@ -1133,6 +1136,8 @@
   //.
   //. The Either type satisfies the [Semigroup][], [Monad][], and [Extend][]
   //. specifications.
+  //.
+  //. This type is exported as `S.EitherType` for use with [sanctuary-def][].
```

Comment:
I'm not really sure what to write for the last bit there, it seems like I'd just be repeating the first paragraph.